README.md

```diff
@@ -1 +1,5 @@
-# QueensN
\ No newline at end of file
+# QueensN
+# Code contained within ChessBoard
+# - takes a int number - lets call N
+# - produces chessboards, which are size NxN, filled with N QueensN
+# - to adjust the number of given queens, use the code block at the end of chessboards (current N = 6)
```

Comment:
this is helpful info to define what this code does, but additionally I think it would be good to add steps to run this program locally